R/routes.R

```diff
@@ -80,7 +80,7 @@
 #' route_cyclestreet(c(-2, 52), c(-1, 53), "fastest")
 #' }
 route_cyclestreet <- function(from, to, plan = "fastest", silent = TRUE, pat = cyclestreet_pat(),
-                              base_url = "http://www.cyclestreets.net/api/", reporterrors = FALSE){
+                              base_url = "http://www.cyclestreets.net", reporterrors = FALSE){
```

Comment:
The main rationale for having the base URL as a parameter is to allow users to run their own instance locally. If you're going to be running hundreds of thousands or millions of requests in a short time period you would probably go over your quota very quickly (and it would be much faster locally).